easybuild/easyconfigs/p/Python/Python-2.7.12-intel-2016b.eb

```diff
@@ -18,7 +18,8 @@ dependencies = [
     ('libreadline', '6.3'),
     ('ncurses', '6.0'),
     ('SQLite', '3.13.0'),
-    ('Tk', '8.6.5'),
+    # We don't build Tkinter as it requires an X11 stack
+    # ('Tk', '8.6.5'),
```

Comment:
Then all the work is fully blocked. No Mesa, no openfoam, no paraview. This is a major blocker and it needs to go